### ISSUE TITLE
Run LunarLander-v2 experiments as benchmark

### ISSUE THREAD

**Issue by yamatokataoka** [yamatokataoka/reinforcement-learning-replications#12]
experiment stats
- epochs (number of updates policy): 1000
- steps_per_epoch: 4000
- n_value_gradients: 80
- gamma: 0.99
- gae_lambda: 0.97
- value function learning_rate: 1e-3
- policy learning_rate: 3e-4
- seed: (0 ~ 2)
